app/controllers/books_controller.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+class BooksController < ApplicationController
+  before_action :set_book, only: %i[show edit update destroy]
+
+  def index
+    @books = Book.page params[:page]
```

Comment:
missing brackets

Reply:
so, is everyone allowed to view all books from all users?

Reply:
PostgreSQL does not guarantee records ordering
at some point, it can happen that some books appear on several pages